src/raven.js

```diff
@@ -777,11 +777,16 @@ Raven.prototype = {
             var xhrproto = XMLHttpRequest.prototype;
             fill(xhrproto, 'open', function(origOpen) {
                 return function (method, url) { // preserve arity
-                    this.__raven_xhr = {
-                        method: method,
-                        url: url,
-                        status_code: null
-                    };
+
+                    // intentionally checking against 0 here (start of string)
+                    if (url.indexOf(self._globalEndpoint) !== 0) {
```

Comment:
What abotu checking for the public key anywhere int he url? That'd only exist in a url if it were bound for the sentry server since it should effectively be a UUID anyways.